src/shared/AddRoom.js

```diff
@@ -0,0 +1,37 @@
+import Component from '../Component.js';
+import { auth, roomsRef } from '../services/firebase.js';
+
+class AddRoom extends Component {
+    render() {
+        const form = this.renderDOM();
```

Comment:
this block of code in your ```render``` method should be refactored slightly. The whitespace is not consistent with the rest of your ```js``` files and it is hard to read!